src/app/components/Progress/Progress.module.css

```diff
@@ -0,0 +1,10 @@
+.container {
+  display: grid;
+  row-gap: 0.5rem;
+  justify-items: center;
+  background-color: #f0f;
+  border-radius: 20px;
+  width: 200px;
+  height: 50px;
```

Comment:
Do you like to increase the size of the progress if you change the font size? If so, this should be relative

Reply:
this is the box, that includes the progress bar
<img width="292" alt="Bildschirmfoto 2021-11-11 um 15 43 10" src="https://user-images.githubusercontent.com/91462519/141317439-4fb38bc9-86b6-4979-b02d-17726ddcacde.png">